### PR TITLE
Upgrade django version to 2.2

### PR DESCRIPTION
**Pull request by Arun-integrella**

### CODE REVIEW

**Review by davismr — CHANGES_REQUESTED** [review]
Is this for a django 2.2, python 3.7 compatible project?

Update setup.py to reflect correct python version, or jsut drop it.

Tests are not passing:

  File "/Users/mdavis/sites/rottweiler/src/django-rottweiler/rottweiler/view_helpers.py", line 22, in get_all_rules
    for k, v in registry.iteritems():
AttributeError: 'collections.defaultdict' object has no attribute 'iteritems'

**Comment by davismr** [comment]
This is the line that needs fixing:

https://github.com/TwigWorld/django-rottweiler/blob/2caac489d6df3e8bf168b6ab884ec64920879e31/rottweiler/view_helpers.py#L22

**Review by davismr — APPROVED** [review]

File: rottweiler/view_helpers.py [hunk 1]
```diff
@@ -3,7 +3,10 @@
 
 from django.conf import settings
 from django.contrib.auth.models import Permission
-from django.core.urlresolvers import RegexURLPattern, RegexURLResolver
+try:
+    from django.core.urlresolvers import RegexURLPattern as URLPattern, RegexURLResolver as URLResolver
```
Comment (davismr) [inline rottweiler/view_helpers.py:7]: Keep imports on separate lines

add comment to say these are python2, we will need to know which ones to delete later

File: rottweiler/view_helpers.py [hunk 2]
```diff
@@ -51,28 +54,28 @@ def extract_views_from_urlpatterns(urlpatterns, base=''):
...
             except ViewDoesNotExist:
                 continue
-        elif isinstance(p, RegexURLResolver):
+        elif isinstance(p, URLResolver):
             try:
                 patterns = p.url_patterns
             except ImportError:
                 continue
-            views.extend(extract_views_from_urlpatterns(patterns, base + p.regex.pattern))
+            views.extend(extract_views_from_urlpatterns(patterns, base + p.pattern.regex.pattern))
```
Comment (davismr) [inline rottweiler/view_helpers.py:67]: This fails a test in python2, syntax should be:

p.regex.pattern

probably easiest to add a switch PYTHON3 = True in import try at top, and use that throughout to decide which syntax to use

**Review by davismr — CHANGES_REQUESTED** [review]

**Comment by Arun-integrella** [comment]
Superseeded by PR9.

**Pull request closed without merging**